.editorconfig

```diff
@@ -0,0 +1,133 @@
+# editorconfig.org
```

Comment:
I would like to somehow indicate which settings are different from the VS defaults, maybe append a comment to the end of the line like `# vs-default: <default value>`?

Reply:
So far, most of the settings appear to be the VS defaults.

Reply:
Should we simply leave out the VS defaults? At least for the VS specific settings for others that might be shared with other editors with different defaults they are still useful.

Reply:
I considered this, but I think it's best to include them since they may be respected by other tools (e.g. dotnet-format, VS Code), either now or in the future.